From d6d57c9ede6a03128eaae35e210f4f08f9dd059e Mon Sep 17 00:00:00 2001
From: Christian Bruel <christian.bruel@foss.st.com>
Date: Fri, 1 Jul 2022 11:10:48 +0200
Subject: [PATCH] arm64: defconfig: Enable the STM32MP25 PCIe RootComplex mode

PCIe controller Based on DesignWare core and STM32 ComboPHY to generate
the refClk.

Signed-off-by: Christian Bruel <christian.bruel@foss.st.com>
Change-Id: I51471e5e30f8014024743d939bf069d4ea3fe7d1
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/361773
Reviewed-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
Domain-Review: Fabrice GASNIER <fabrice.gasnier@foss.st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/fragment-02-defconfig-addons.config b/arch/arm64/configs/fragment-02-defconfig-addons.config
index 45475b1ecd06..67b26c2fe163 100644
--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -41,6 +41,7 @@ CONFIG_NVMEM_REBOOT_MODE=y
 CONFIG_NVMEM_STM32_ROMEM=y
 CONFIG_NVMEM_STM32_TAMP=y
 CONFIG_OF_OVERLAY=y
+CONFIG_PCIE_STM32=y
 CONFIG_PHY_STM32_COMBOPHY=y
 CONFIG_PHY_STM32_USB2PHY=y
 CONFIG_PWM_STM32=m
-- 
2.39.5

